assets/minecraft/font/default.json

```diff
@@ -4,7 +4,7 @@
       "type": "bitmap",
       "file": "minecraft:emotes/aqua_chibi.png",
       "ascent": 9,
-      "height": 11,
+      "height": 9,
```

Comment:
We actually have a separate config that genereates this one on the server, I'll add the emotes you wanted and possibly lower the size of some others, for now just revert the changes in this file.